.github/actions/retry/action.yml

```diff
@@ -0,0 +1,23 @@
+name: "Retry Steps"
```

Comment:
Can the directory be renamed from `actions` to `modules` please and file to be `retry.yml` so it can be used for other similar instances?